### PR TITLE
Added RedisTimeSeries with Python and Go

### PR DESCRIPTION
**Pull request by ajeetraina**
Added couple of stuffs:
- Started RedisTimeSeries Tutorials
- RedisTimeSeries using Go
- RedisTimeSeries with Python
- Added myself as author
- Added author to various tutorials

### CODE REVIEW

File: docs/howtos/redistimeseries/getting-started/index-redistimeseries.mdx [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+id: index-gettingstarted
+title: Getting Started with RedisTimeSeries 
+sidebar_label: Getting Started with RedisTimeSeries 
+slug: /howtos/redistimeseries/getting-started
+authors: [ajeet]
+---
+
+
+[RedisTimeseries](https://redis.com/modules/redis-timeseries/) is a Redis module developed by Redis Labs to enhance your experience managing time-series data with Redis. It simplifies the use of Redis for time-series use cases such as internet of things (IoT) data, stock prices, and telemetry. With RedisTimeSeries, you can ingest and query millions of samples and events at the speed of Redis. Advanced tooling such as downsampling and aggregation ensure a small memory footprint without impacting performance. Use a variety of queries for visualization and monitoring with built-in connectors to popular monitoring tools like Grafana, Prometheus, and Telegraf.
```
Comment (simonprickett) [inline docs/howtos/redistimeseries/getting-started/index-redistimeseries.mdx]: @ajeetraina this should be "developed by Redis".

File: docusaurus.config.js [hunk 2]
```diff
@@ -24,8 +24,14 @@ module.exports = {
         link: 'https://twitter.com/SuzeShardlow',
         title: 'Developer Community Manager',
         image: 'profile_pic_suze_shardlow.jpg'
-      }
+      },
+      ajeet: {
+        name: 'Ajeet Raina',
+        link: 'https:/twitter.com/ajeetsraina',
+        title: 'Developer Relations Manager',
```
Comment (simonprickett) [inline docusaurus.config.js]: @ajeetraina shouldn't this be "Senior Technical Marketing Manager"?

**Review by simonprickett — COMMENTED** [review]
Left some initial comments, will have a look through timeseries article shortly

File: docs/howtos/redistimeseries/getting-started/index-redistimeseries.mdx (hunk 1 above)
Comment (ajeetraina) [reply docs/howtos/redistimeseries/getting-started/index-redistimeseries.mdx]: Yes, a good catch. It should be "Redis".

File: docusaurus.config.js (hunk 2 above)
Comment (ajeetraina) [reply docusaurus.config.js]: Hi @simonprickett Trying to keep it consistent throughout as per https://redis.com/blog/author/ajeet-raina/ But anyway I will go ahead and change it accordingly

Comment (ajeetraina) [reply docusaurus.config.js]: Fixed.

**Review by simonprickett — APPROVED** [review]
Tested locally, looks good to me, will merge.

**Pull request merged**